pontoon/messaging/templates/messaging/includes/sent.html

```diff
@@ -0,0 +1,39 @@
+<div id="sent">
+    {% if sent_messages|length == 0 %}
+    <div class="no">
+        <i class="icon fa fa-envelope fa-fw"></i>
+        <p class="title">No messages sent yet.</p>
+    </div>
+    {% endif %}
+    <ul>
+        {% for message in sent_messages %}
+        <li class="message{% if message.is_new() %} new{% endif %}">
+            <div class="user-avatar">
+                <a href="{{ url('pontoon.contributors.contributor.username', message.sender.username) }}" target="_blank" rel="noopener noreferrer">
+                    <img src="{{ message.sender.gravatar_url(88) }}" height="44" width="44" class="rounded">
+                </a>
+            </div>
+            <div class="details">
+                <div class="info">
+                    <a class="sender" href="{{ url('pontoon.contributors.contributor.username', message.sender.username) }}" target="_blank" rel="noopener noreferrer">{{ message.sender.name_or_email }}</a>
+                    <span class="recipients">
+                    {% with count=message.recipients.count() %}
+                    to {{ count }} recipient{{ count|pluralize }}
+                    {% endwith %}
+                    </span>
+                    <span class="types">
+                    {% if message.notification %}<i class="type far fa-bell" title="Sent as Notification"></i>{% endif %}
+                    {% if message.email %}<i class="type far fa-envelope" title="Sent as Email"></i>{% endif %}
+                    </span>
+                </div>
+                <p class="subject">{{ message.subject }}</p>
+                <div class="body"><div class="value">{{ message.body|safe }}</div></div>
+            </div>
+            <div class="footer">
+                <a href="{{ url('pontoon.messaging.edit_as_new', message.pk) }}" class="edit-as-new"><i class="fa fa-pencil-alt"></i>Edit as new</a>
+                <time>{{ message.sent_at|format_for_inbox }}</time>
```

Comment:
I initially didn't see this "sent at" info at all. I would expect for this to be near the top of the message, where we also have the sender, the recipient count & message type.

Reply:
It's common pattern to show timestamp under the message bubble in messaging apps. I think you'll get used to it quickly.

It'll be busy in the header.